default.yml

```diff
@@ -1,7 +1,8 @@
 require: rubocop-rspec
 
 AllCops:
-  TargetRubyVersion: 2.4
+  NewCops: enable
+  TargetRubyVersion: 2.7
```

Comment:
We might need to find all the repos using `percy-style` and see what happens when we run this change in each of them 🤔

Reply:
IDK whether all consuming places pin their gem version. If they do, this hopefully will be good as-is, though.

Reply:
I see you're doing this for api here https://github.com/percy/percy-api/pull/2786/files ❤️

Reply:
Ah, I realize I can remove this line from here because each project still has its own rubocop config 🤷🏾‍♂️